pkg/services/serviceaccounts/serviceaccounts.go

```diff
@@ -9,7 +9,9 @@ import (
 type Service interface {
 	CreateServiceAccount(ctx context.Context, saForm *CreateServiceaccountForm) (*models.User, error)
 	DeleteServiceAccount(ctx context.Context, orgID, serviceAccountID int64) error
+	HasServiceAccountsMigrated(ctx context.Context, orgID int64) bool
```

Comment:
Suggestion:

```suggestion
	Migrated(ctx context.Context, orgID int64) bool
```

reads as `(serviceaccounts.Service).Migrated(…)`. Alternatively `HasMigrated` is also more concise and conveys the same information.

Reply:
gotcha! 👍